frontend/packages/console-app/src/components/nodes/node-dashboard/queries.ts

```diff
@@ -0,0 +1,131 @@
+import * as _ from 'lodash';
+import { QueryWithDescription } from '@console/shared/src/components/dashboard/utilization-card/UtilizationItem';
+
+export enum NodeQueries {
```

Comment:
Since the Bare Metal Hosts use node exporter queries as well, we should eventually update Bare Metal Host dashboard to use these common queries defined here instead of redefining them specifically for BMHs. (see `metal3-plugin/src/components/baremetal-hosts/dashboard/queries.ts`).